### PR TITLE
ENH: add sliders and c-phi for etapip auto notebook

### PR DESCRIPTION
**Pull request by shenvitor**

### CODE REVIEW

File: docs/eta-pi-p/automated.ipynb [hunk 1]
```diff
@@ -366,34 +324,174 @@
...
+    "])\n",
+    "tab_a2_content = w.VBox([\n",
+    "    w.HBox(categorized_sliders_m[2] + categorized_sliders_gamma[2]),\n",
+    "    w.VBox(categorized_cphi_pair[2]),\n",
+    "])\n",
+    "\n",
+    "UI = w.Tab([tab_N_content, tab_D_content, tab_a2_content])\n",
+    "UI.set_title(0, \"N*\")\n",
+    "UI.set_title(1, \"Δ*\")\n",
+    "UI.set_title(2, \"a₂*\")"
```
Comment (redeboer) [inline docs/eta-pi-p/automated.ipynb:412]: ChatGPT. Why not use `titles=["N*", "Δ*", "a₂*"]` as in the other notebook?

Comment (redeboer) [reply docs/eta-pi-p/automated.ipynb:412]: Also, what is $a_2^*$?

File: docs/eta-pi-p/automated.ipynb [hunk 2]
```diff
@@ -366,34 +324,174 @@
...
+    "tab_D_content = w.VBox([\n",
+    "    w.HBox(categorized_sliders_m[1] + categorized_sliders_gamma[1]),\n",
+    "    w.VBox(categorized_cphi_pair[1]),\n",
+    "])\n",
+    "tab_a2_content = w.VBox([\n",
+    "    w.HBox(categorized_sliders_m[2] + categorized_sliders_gamma[2]),\n",
+    "    w.VBox(categorized_cphi_pair[2]),\n",
+    "])\n",
+    "\n",
+    "UI = w.Tab([tab_N_content, tab_D_content, tab_a2_content])\n",
```
Comment (redeboer) [inline docs/eta-pi-p/automated.ipynb:409]: This could be combined into a loop over `range(3)`

File: docs/eta-pi-p/automated.ipynb [hunk 3]
```diff
@@ -331,10 +292,7 @@
     },
     "mystnb": {
      "code_prompt_show": "Collect resonances for each subsystem"
-    },
-    "tags": [
-     "hide-input"
-    ]
+    }
```
Comment (redeboer) [inline docs/eta-pi-p/automated.ipynb:295]: Not hidden anymore?

File: docs/eta-pi-p/automated.ipynb [hunk 4]
```diff
@@ -183,9 +182,6 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
```
Comment (redeboer) [inline docs/eta-pi-p/automated.ipynb:184]: Try make these hidden again / revert these

**Review by redeboer — CHANGES_REQUESTED** [review]

File: docs/eta-pi-p/automated.ipynb (hunk 4 above)
Comment (redeboer) [reply docs/eta-pi-p/automated.ipynb]: For reference:
```shell
git checkout origin/main -- docs/eta-pi-p/automated.ipynb
git restore --staged docs/eta-pi-p/automated.ipynb
git add -p docs/eta-pi-p/automated.ipynb
# now only stage the hunks from the main branch that you want to revert
```

File: docs/eta-pi-p/automated.ipynb (hunk 1 above)
Comment (shenvitor) [reply docs/eta-pi-p/automated.ipynb:421]: https://pdglive.lbl.gov/ParticleGroup.action?init=0&node=MXXX005

because there are two a₂:  (1320) and (1700)

**Review by redeboer — APPROVED** [review]
Splendid 🌟

**Pull request merged**